lib/spack/spack/build_systems/gnu.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os.path
+
+import spack.package
+
+
+class GNUPackage(spack.package.PackageBase):
```

Comment:
I would call this `GNUMirrorPackage` or something indicating that it's not just some piece of GNU software.

Reply:
Done